src/dist/mod.rs

```diff
@@ -1,17 +1,1339 @@
 //! Installation from a Rust distribution server
 
-pub use crate::dist::notifications::Notification;
-pub use dist::*;
-
-pub mod temp;
+use crate::{
```

Comment:
The imports look a bit disorganized. I think we should group as follows:

* `std` imports
* external crate imports
* `crate` imports (from outside this module only)
* Module declarations, each grouped with imports from that module

Reply:
Done, and thanks for the hint :)